api/app/src/main/kotlin/packit/repository/PacketGroupDisplayRepository.kt

```diff
@@ -0,0 +1,11 @@
+package packit.repository
+
+import org.springframework.security.access.prepost.PostFilter
+import packit.model.PacketGroupDisplay
+
+interface PacketGroupDisplayRepository : ViewRepository<PacketGroupDisplay, String> {
```

Comment:
string should be int - left over from when i was using name as the identifier rather than packet group id.